src/constants.js

```diff
@@ -44,6 +44,16 @@ export const RESPONSE_CODE_FAIL = 400;
 export const PLACE_ORDER_RESPONSE = 'PLACE_ORDER_RESPONSE';
 export const CLEAR_ORDER_RESPONSES = 'CLEAR_ORDER_RESPONSES';
 
+/** REPORTS */
+export const FETCH_REPORTS_DATA = 'FETCH_REPORTS_DATA';
+export const UPDATE_REPORTS_DATA = 'UPDATE_REPORTS_DATA';
+// Reports range // WHY CAN THESE NOT BE USED?
+// export const RANGE_CURRENT_WEEK = 'currentweek';
+// export const RANGE_PREVIOUS_WEEK = 'previousweek';
+// export const RANGE_CURRENT_YEAR = 'currentyear';
+// export const RANGE_PREVIOUS_YEAR = 'previousyear';
+
+
 // old
```

Comment:
If old, are there lines we should take out? Also consider removing commented lines above if unused